lib/std/event/loop.zig

```diff
@@ -541,8 +565,8 @@ pub const Loop = struct {
             .data = 0,
             .udata = @ptrToInt(&resume_node.base),
         };
-        const kevent_array = (*const [1]os.Kevent)(&kev);
-        const empty_kevs = ([*]os.Kevent)(undefined)[0..0];
+        const kevent_array = @ptrCast(*const [1]os.Kevent, &kev);
```

Comment:
To maintain the previous semantics: 

```suggestion
        const kevent_array = @as(*const [1]os.Kevent, &kev);
```

Or you could make `kev` above an array of size 1 and use the address-of operator below in the kevent() call.

Same deal with the other `@ptrCast` usage here. `@ptrCast` is scary! Try not to use it!